CHANGES.rst

```diff
@@ -8,6 +8,7 @@ Unreleased
 
 - Fixed translations. French now works again.
 - Fixed missing whitespace.
+- Align with license change. Enterprise features are now available to everyone.
```

Comment:
We normally put most recent changes on top. While on that could you also add empty lines between each entry please? thx!

Reply:
This has been addressed, thanks.